test/e2e/e2ethanos/services.go

```diff
@@ -662,7 +662,14 @@ func NewMemcached(name string) *e2e.ConcreteService {
 	return memcached
 }
 
-func NewToolsBucketWeb(name string, bucketConfig client.BucketConfig, routePrefix, externalPrefix string) (*Service, error) {
+func NewToolsBucketWeb(
+	name string,
+	bucketConfig client.BucketConfig,
+	routePrefix,
+	externalPrefix string,
+	minTime string,
+	maxTime string,
```

Comment:
I feel using time.Time is better than string. We can convert them to RFC3339 format below

Reply:
but it it the [pkg/model/TimeOrDurationValue type](https://github.com/thanos-io/thanos/blob/main/pkg/model/timeduration.go#L14), could be absolute time or relevant time duration, but the string value `(*TimeOrDurationValue.)String()` can not be parsed at `(*TimeOrDurationValue.).Set(string)`, so just use string for simply.

Reply:
That makes sense. 💪